doc/api-asyncio/asyncio_gridfs.rst

```diff
@@ -235,7 +235,9 @@ Store blobs of data in `GridFS <http://dochub.mongodb.org/core/gridfs>`_.
           async def upload():
               my_db = AsyncIOMotorClient().test
               fs = AsyncIOMotorGridFSBucket(my_db)
-              async with await fs.new_file() as gridin:
+              async with await fs.open_upload_stream(
+                  "test_file", chunk_size_bytes=4, 
+                  metadata={"contentType": "text/plain"}) as gridin:
```

Comment:
I know this is copied from pymongo docs but can you remove the `chunk_size_bytes=4` parameter? It's bad practice to set such a low chunk size. I'll remove this from pymongo's docs too.

Reply:
Sure thing!  I agree to the small chunk size being bad practice.   I removed the `chunk_size_bytes=4` in the whole document.   Is that what you were wanting, or just for this code example?